crates/matrix-sdk/src/event_cache/store.rs

```diff
@@ -251,12 +76,17 @@ impl RoomEvents {
         self.chunks.push_items_back(events)
     }
 
+    /// Push a gap after existing events.
```

Comment:
I use the word _events_ in `RoomEvents` and _items` in `LinkedChunk`. In `LinkedChunk`, an item can be anything, but in `RoomEvents`, the items of `LinkedChunk` _are_ events.

Reply:
Theoretically, the last item of `self.chunks` could be a gap, so we could be pushing a gap after a gap? (even though we *should not*) In this case, `after existing events` would be misleading, because the pushed gap would go after a gap, not after an event.

Reply:
It's possible to have multiple contiguous gap yes. I'll update the doc.